app/controllers/idv/in_person/address_search_controller.rb

```diff
@@ -14,27 +14,38 @@ def index
 
       def addresses(search_term)
         addresses = geocoder.find_address_candidates(SingleLine: search_term).slice(0, 1)
+        if addresses.empty?
+          analytics.idv_in_person_locations_searched(
+            success: false, errors: 'No address candidates found by arcgis',
+          )
+        end
         { json: addresses, status: :ok }
-      rescue Faraday::ConnectionFailed => err
-        analytics.idv_arcgis_request_failure(
-          api_status_code: 422,
-          exception_class: err.class,
-          exception_message: err.message,
-          response_body_present: err.respond_to?(:response_body) && err.response_body.present?,
-          response_body: err.respond_to?(:response_body) && err.response_body,
-          response_status_code: err.respond_to?(:response_status) && err.response_status,
-        )
-        { json: [], status: :unprocessable_entity }
-      rescue Faraday::TimeoutError => err
-        analytics.idv_arcgis_request_failure(
-          api_status_code: 422,
+
+      rescue StandardError => err
+        apiError = err.instance_of?(Faraday::ClientError) ? err.response['error'] : nil
```

Comment:
@tomas-nava this change is to address your 1st comment. i noticed there's a method in Geocoder called `handle_api_errors` that throws a client error when the response has an error code. this should log when that happens.